.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Questions
-    url: https://github.com/orgs/ublue-os/discussions/
-    about: Ask a question, share tips, and help others
\ No newline at end of file
+  - name: Questions and Feedback
+    url: https://universal-blue.discourse.group/c/bluefin/6
```

Comment:
This link seems to point to the bluefin category?

Reply:
Ooops, fixed!